frontend/src/app/projects/models/backup-settings/trigger-pipeline/trigger-pipeline.component.html

```diff
@@ -30,110 +20,81 @@
+            <div class="flex flex-wrap items-center justify-between gap-y-1">
+              <span style="flex-basis: 45%"
+                ><b>TeamForCapella</b> <br />
+                Instance <i>{{ pipeline.t4c_model.instance_name }}</i
+                >, <br />Repository
+                <i>{{ pipeline.t4c_model.repository_name }}</i
+                >, <br />Project:
+                <i>{{ pipeline.t4c_model.project_name }}</i> </span
+              ><span style="flex-basis: 10%"
+                ><mat-icon class="mat-icon-position top"
+                  >arrow_forward</mat-icon
+                >
+              </span>
+              <span style="flex-basis: 45%">
+                <b>Git</b> <br />
+                URL: <i>{{ pipeline.git_model.path }}</i
+                >, <br />revision: <i>{{ pipeline.git_model.revision }}</i
+                >, <br />entrypoint:
+                <i>{{ pipeline.git_model.entrypoint }}</i>
               </span>
-              <hr />
-              <div class="flex flex-wrap items-center justify-between gap-y-1">
-                <span style="flex-basis: 45%"
-                  ><b>TeamForCapella</b> <br />
-                  Instance <i>{{ pipeline.t4c_model.instance_name }}</i
-                  >, <br />Repository
-                  <i>{{ pipeline.t4c_model.repository_name }}</i
-                  >, <br />Project:
-                  <i>{{ pipeline.t4c_model.project_name }}</i> </span
-                ><span style="flex-basis: 10%"
-                  ><mat-icon class="mat-icon-position top"
-                    >arrow_forward</mat-icon
-                  >
-                </span>
-                <span style="flex-basis: 45%">
-                  <b>Git</b> <br />
-                  URL: <i>{{ pipeline.git_model.path }}</i
-                  >, <br />revision: <i>{{ pipeline.git_model.revision }}</i
-                  >, <br />entrypoint:
-                  <i>{{ pipeline.git_model.entrypoint }}</i>
-                </span>
-              </div>
-            </button>
-          }
+            </div>
+            <hr class="my-2" />
+            <div class="flex flex-wrap gap-2">
+              <button
+                mat-raised-button
+                (click)="runPipeline(pipeline)"
+                color="primary"
+              >
+                <mat-icon>play_circle_filled</mat-icon>
+                Run pipeline
+              </button>
+              <button
+                mat-raised-button
+                (click)="openPipelineRuns(pipeline)"
+                color="primary"
+                type="button"
+              >
+                <mat-icon>list</mat-icon>
+                View runs
+              </button>
+              <button
+                mat-raised-button
+                (click)="removePipeline(pipeline)"
+                color="warn"
+              >
+                <mat-icon>delete</mat-icon>
+                Remove pipeline
+              </button>
```

Comment:
Consider moving the remove button to the left. Ideally, the rightmost button should be the primary action (in this case probably the "Run Pipeline" button).

Reply:
Good hint. Changed it, the order is now: 
![image](https://github.com/user-attachments/assets/d163f8eb-f11d-4b6c-b92f-2fc192a2ddc9)